### PR TITLE
DOC Adding API and fixing some urls

### PR DESCRIPTION
**Pull request by jmmovilla**
@hmunfru @flopezag @jframos

### CODE REVIEW

File: doc/user_guide.rst [hunk 1]
```diff
@@ -57,10 +57,251 @@ For the rest of the explanation, we are going to configure a set of variables:
...
+
+Next we detail some operations that can be done in the catalogue managemente api regarding the Abstract Environments.
+Abstract Environments are environments defined by the administrator. They are available for all FIWARE users.
+
+
+**Get the Abstract Environment list from the catalogue**
+
+.. code::
+
+    $ curl -v -H "Content-Type: application/json" -H "Accept: application/json" -H
```
Comment (jframos) [inline doc/user_guide.rst]: Body and response examples are in XML representation. Check Content-Type and Accept headers in curl requests of this document.

File: doc/user_guide.rst [hunk 2]
```diff
@@ -57,10 +57,251 @@ For the rest of the explanation, we are going to configure a set of variables:
...
+				<affinity>None</affinity>
+				<region>Spain</region>
+ 			</tierDtos>
+			<name>orion</name>
+			<description>Environment orion</description>
+ 		</environmentDto>
+ 		...
+   </environmentDtoes>
+
+** Get a particular Abstract Environment
```
Comment (jframos) [inline doc/user_guide.rst]: RST mistake?

File: doc/user_guide.rst [hunk 3]
```diff
@@ -57,10 +57,251 @@ For the rest of the explanation, we are going to configure a set of variables:
...
+			<icono>http://blog.theinit.com/wp-content/uploads/2012/03/bc358_MongoDB.png</icono>
+			<securityGroup />
+			<keypair />
+			<floatingip>false</floatingip>
+			<affinity>None</affinity>
+			<region>Spain</region>
+ 		</tierDto>
+	</tierDtoes>
+
+** Get a particular Tier associated to a Abstract Environment
```
Comment (jframos) [inline doc/user_guide.rst]: RST mistake?

File: doc/user_guide.rst [hunk 4]
```diff
@@ -57,10 +57,251 @@ For the rest of the explanation, we are going to configure a set of variables:
...
+		<name>{tier-name}</name>
+		<image>0dbf8aff-5dc5-4d6c-9f9c-1e6801e0b629</image>
+		<flavour>2</flavour>
+		<keypair>jesusmmovilla57</keypair>
+		<floatingip>false</floatingip>
+		<region>Spain</region>
+	</tierDto> 
+
+
+** Delete a particular Tier associated to a Abstract Environment
```
Comment (jframos) [inline doc/user_guide.rst]: RST mistake?

**Comment by hmunfru** [comment]
LGTM

**Pull request merged**